suspenders.gemspec

```diff
@@ -30,6 +30,10 @@ rush to build something amazing; don't use it if you like missing deadlines.
   s.add_dependency 'bitters', '~> 1.2'
   s.add_dependency 'bundler', '~> 1.3'
   s.add_dependency 'rails', Suspenders::RAILS_VERSION
+  s.add_dependency 'simple_form', '~> 3.2'
+  s.add_dependency 'title', '~> 0.0'
+  s.add_dependency 'quiet_assets', '~> 1.1'
+  s.add_dependency 'capybara-webkit', '~> 1.8'
```

Comment:
I think these (and bitters) should be `development_dependency`s (used for running tests). Generated app's Gemfile will take care of those while using suspenders.